examples/one-page/index.html

```diff
@@ -83,25 +68,24 @@
       `;
 
       const Presentation = () => html`
-        <${Deck} theme=${theme} template=${template}>
-          <${Slide}>
-            <${FlexBox} height="100%">
-              <${SpectacleLogo} size=${500} />
-            </${FlexBox}>
-            <${Notes}>Spectacle supports notes per slide.<ol>
-                <li>Notes can now be HTML markup!</li>
-                <li>Lists can make it easier to make points.</li>
-              </ol>
-            </${Notes}>
-          </${Slide}>
-          <${Slide}>
-            <${FlexBox} height="100%" flexDirection="column">
-              <${Heading} margin="0px" fontSize="150px">✨<i>Spectacle</i> ✨</${Heading}>
-              <${Heading} margin="0px" fontSize="h2">A ReactJS Presentation Library</${Heading}>
-              <${Heading} margin="0px 32px" color="primary" fontSize="h3">Where you can write your decks in JSX, Markdown, or MDX!</${Heading}>
-            </${FlexBox}>
-          </${Slide}>
-          <${Slide} transition=${{
+        <${Deck} theme=${theme} template=${html`<${DefaultTemplate} />`}><${Slide}>
```

Comment:
None of this changed other than the `template` prop, but the generation script is keeping the `<${Slide}>` tag same line thus changing all the following indentation. Since this is auto-generated, it's a lower concern.